.circleci/config.yml

```diff
@@ -315,10 +311,10 @@ commands: # reusable commands with parameters
       - run:
           name: Run Rails tests
           command: |
+            list=/tmp/test-list-$$
             taskname=$(echo $CIRCLE_JOB | sed -e 's/-\(postgres\|oracle\|[0-9]\).*//')
-            TESTS=$(bundle exec rake "test:files:${taskname}" | circleci tests split --split-by=timings)
-            echo "bundle exec rake test:run  TESTOPTS=--verbose --verbose --trace TEST='${TESTS}'"
-            bundle exec rake test:run TEST="${TESTS}" TESTOPTS=--verbose --verbose --trace
+            bundle exec rake "test:files:${taskname}" | circleci tests run --command="cat > $list" --verbose --split-by=timings
+            bundle exec rake test:run TEST="$(cat $list)" TESTOPTS=--verbose --verbose --trace
```

Comment:
I don't understand this one, shouldn't it be like this?

```
TESTS=$(bundle exec rake "test:files:${taskname}" | circleci tests split --split-by=timings)
circleci tests run --command="bundle exec rake test:run TEST='${TESTS}' TESTOPTS=--verbose --verbose --trace" --verbose
```

Reply:
Splitting should be done by `circleci tests run`, see the document linked in description. We are following documentation. Although I don't know if latest fixes are in place yet. See circleci/circleci-docs#8075